### PR TITLE
Ansible module

### PR DESCRIPTION
**Pull request by artewka**

### CODE REVIEW

File: ansible/dbsrv.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+- name: Install software to web-srv
+  hosts: db_servers
+  become: yes
+
+  tasks:
+  - name: Add GPG key for maria_db
+    shell: apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8
```
Comment (DmytroMorhulDevPro) [inline ansible/dbsrv.yml]: please use ansible modules here

File: ansible/dbsrv.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  - name: Add MDB repo
+    shell: add-apt-repository 'deb [arch=amd64,arm64,ppc64el] http://mirrors.accretive-networks.net/mariadb/repo/10.3/ubuntu bionic main'
```
Comment (DmytroMorhulDevPro) [inline ansible/dbsrv.yml]: please use ansible modules here

File: ansible/dbsrv.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    apt:
+      name: mariadb-server
+      state: latest
+
+  - name: Install phpmyadmin
+    apt:
+      name: phpmyadmin
+      state: latest
+
+  - name: Get php version
```
Comment (DmytroMorhulDevPro) [inline ansible/dbsrv.yml:26]: please use ansible modules here

File: ansible/dbsrv.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    shell: echo "<?php phpinfo(); ?>" > /var/www/html/phpinfo.php
+
+  - name: Create_symb_link for phpmyadm
+    shell: ln -s /usr/share/phpmyadmin /var/www/html/phpmyadmin
```
Comment (DmytroMorhulDevPro) [inline ansible/dbsrv.yml]: please use ansible modules here

File: ansible/websrv.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      name: nginx
+      state: latest
+  
+  - name: Install php
+    apt:
+      name: php-fpm
+      state: latest
+  
+  - name: Get php version
+    shell: echo "<?php phpinfo(); ?>" > /var/www/html/phpinfo.php
```
Comment (DmytroMorhulDevPro) [inline ansible/websrv.yml]: please use ansible modules here

**Comment by artewka** [comment]
done, check please

**Review by DmytroMorhulDevPro — APPROVED** [review]

**Pull request merged**